smac/smac_cli.py

```diff
@@ -59,29 +60,32 @@ def main_cli(self):
                         in_runhistory_fn_list=args_.warmstart_runhistory,
                         cs=scen.cs,
                         aggregate_func=aggregate_func)
-            
+
         initial_configs = None
         if args_.warmstart_incumbent:
             initial_configs = [scen.cs.get_default_configuration()]
             for traj_fn in args_.warmstart_incumbent:
                 trajectory = TrajLogger.read_traj_aclib_format(fn=traj_fn, cs=scen.cs)
                 initial_configs.append(trajectory[-1]["incumbent"])
-                
+
         if args_.modus == "SMAC":
             optimizer = SMAC(
-                scenario=scen, 
-                rng=np.random.RandomState(args_.seed), 
-                runhistory=rh, 
+                scenario=scen,
+                rng=np.random.RandomState(args_.seed),
+                runhistory=rh,
                 initial_configurations=initial_configs)
         elif args_.modus == "ROAR":
             optimizer = ROAR(
-                scenario=scen, 
-                rng=np.random.RandomState(args_.seed), 
-                runhistory=rh, 
+                scenario=scen,
+                rng=np.random.RandomState(args_.seed),
+                runhistory=rh,
                 initial_configurations=initial_configs)
         try:
             optimizer.optimize()
-
+        except TAEAbortException:
```

Comment:
Does this also catch the exception that the first run crashed? If yes, the error message would be incorrect (as the tae and not the ta signaled to abort).